src/main/java/ro/msg/learning/shop/configuration/TestSecurityConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package ro.msg.learning.shop.configuration;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+
+@Configuration
+@EnableWebSecurity
+@AllArgsConstructor
+@Profile("test")
+@Slf4j
+public class TestSecurityConfiguration {
+
+	@Bean
+	protected SecurityFilterChain configureTestSecurity(HttpSecurity httpSecurity) throws Exception {
```

Comment:
I understand that this config is used for tests. But you need to integrate the authentication into test in order to properly test the application. So the tests should use authentication (you can create a test user in the database at startup when the database is created). Then, the requests should contain the credentials for that user.

Reply:
done.